ubuntudesign/www.ubuntu.com
# Mobile structure

## Done

 - Added new site navigation for /mobile
 - Created stub pages for all of the mobile sections
 - Updated redirects to point /phone and /tablet URLs to /mobile
 - Removed tablet section
 - Added /mobile into full-width css configuration sections

## QA

 - go to /mobile and see that it exists as a contentless stub
 - check the secondary nav and that all of the sections work
 - go to /phone or /tablet and see the redirect to /mobile
 - Try some deeper pages from live ubuntu.com/phone or /tablet and check for sensible redirects.
